server/db/index.js

```diff
@@ -9,13 +11,45 @@ connection.once('open', () => {
   console.log('successfully connected to mongoose');
 });
 
-module.exports.getInfo = async (gameId, callback) => {
-  await connection;
-  connection.db.collection('gameCarouselInfo', async (err, collection) => {
-    if (err) {
-      callback(err);
+const gameCarouselInfo = new mongoose.Schema({
+  game_id: {
+    type: Number,
+    unique: true,
+  },
+  category_tree: {
+    genres: {
+      genre_name: Array,
+    },
+  },
+  genre: String,
+  game_title: String,
+  video_photo_carousel: Array,
+  game_photo: String,
+  short_description: String,
+  recent_reviews: String,
+  recent_reviews_count: Number,
+  all_reviews: String,
+  all_reviews_count: Number,
+  release_date: Date,
+  developer: String,
+  publisher: String,
+  popular_tags: Array,
+},
+{ collection: 'gameCarouselInfo' });
+
+const GameCarouselInfo = mongoose.model('GameCarouselInfo', gameCarouselInfo);
+
+const getInfo = async (gameId, callback) => {
+  GameCarouselInfo.find({ game_id: gameId }, (err, gameData) => {
+    if (err || !gameData.length) {
```

Comment:
Really clean code here, looks solid! I'm also not familiar with some of this Mongoose syntax such as "exec()" and "toArray()", already have an idea of how these work but I'll definitely be looking up what these do!